database/migrations/2020_01_15_193100_create_bibliomanuel_table.php

```diff
@@ -0,0 +1,79 @@
+<?php
+use App\Traits\MigrationHelper;
+use Carbon\Carbon;
+use Illuminate\Database\Migrations\Migration;
+use Illuminate\Database\Schema\Blueprint;
+
+class CreateBibliomanuelTable extends Migration
+{
+    use MigrationHelper;
+
+    /**
+     * Run the migrations.
+     *
+     * @return void
+     */
+    public function up()
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Avoid using short method names like CreateBibliomanuelTable::up(). The configured minimum method name length is 3.](https://app.codacy.com/manual/scriptotek/ub-baser/pullRequest?prid=4823507)